app/assets/stylesheets/mixins/default-colors.scss

```diff
@@ -58,10 +58,10 @@ $yellow:      #D1C905;
 
 $cover-photo-url: "cover_photos/header/default.jpg";
 $small-cover-photo-url: "cover_photos/header/default.jpg";
-$wide-logo-lowres-url: "logos/full/default.png";
-$wide-logo-highres-url: "logos/full/default-highres.png";
-$square-logo-lowres-url: "logos/mobile/default.png";
-$square-logo-highres-url: "logos/mobile/default-highres.png";
+$wide-logo-lowres-url: "";
+$wide-logo-highres-url: "";
+$square-logo-lowres-url: "";
+$square-logo-highres-url: "";
```

Comment:
If these are not cleared, the default Sharetribe logo shows up when the image(s) are uploaded, but the SCSS compilation has not yet finished.